### PR TITLE
Add fields to feedback page

### PR DESCRIPTION
**Pull request by doctorpod**
Add fields to feedback page. Originally #192 which was merged by mistake.

### CODE REVIEW

File: app/models/concerns/date_of_birth.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  extend ActiveSupport::Concern
+
+  MAX_AGE = 120
+
+  included do
+    validates :date_of_birth,
+      allow_blank: true,
+      inclusion: {
+        in: ->(d) { d.minimum_date_of_birth..d.maximum_date_of_birth },
+        message: "Must be less than #{MAX_AGE} years ago"
```
Comment (doctorpod) [inline app/models/concerns/date_of_birth.rb]: Translate! (Suggested by Alan K)

**Comment by kyliehavelock** [comment]
screen shot 2017-05-30 at 15 19 18

The month validation is a bit strange

File: spec/features/submit_feedback_spec.rb [hunk 2]
```diff
@@ -1,26 +1,50 @@
 require 'rails_helper'
 
 RSpec.feature 'Submit feedback', js: true do
-  before do skip 'Features specs not yet fixed' end
```
Comment (alan) [inline spec/features/submit_feedback_spec.rb]: Nice!

File: app/models/concerns/person.rb [hunk 3]
```diff
@@ -22,7 +20,7 @@ def anonymized_name
   end
 
   def minimum_date_of_birth
```
Comment (alan) [inline app/models/concerns/person.rb]: Is this method here needed since it is the new concern?

File: lib/maybe_date.rb [hunk 4]
```diff
@@ -13,12 +13,20 @@ def coerce(value)
     if value.is_a?(String)
       Date.parse(value)
     elsif value.respond_to?(:values_at)
-      ymd = value.values_at(:year, :month, :day).map(&:to_i)
-      begin
-        Date.new(*ymd)
-      rescue ArgumentError # e.g. invalid date such as 2010-14-31
-        UncoercedDate.new(*ymd)
-      end
+      coerce_with_values_at(value)
```
Comment (alan) [inline lib/maybe_date.rb]: Can we add specs to this? Also, if this is the same as in pvb2 we should extract it out into a gem

File: spec/features/submit_feedback_spec.rb [hunk 5]
```diff
@@ -1,26 +1,50 @@
...
-
   include FeaturesHelper
 
-  scenario 'submitting feedback' do
+  before do
+    allow(PrisonVisits::Api.instance).to receive(:create_feedback)
```
Comment (alan) [inline spec/features/submit_feedback_spec.rb]: Why do you need to stub this api, if we are using vcr then it shouldn't be stubbed

**Pull request merged**